gliderpy/plotting.py

```diff
@@ -0,0 +1,58 @@
+"""Some convenience functions to help visualize glider data."""
+
+from __future__ import annotations
+
+import warnings
+from typing import TYPE_CHECKING
+
+try:
+    import cartopy.crs as ccrs
+    import matplotlib.dates as mdates
+    import matplotlib.pyplot as plt
+except ModuleNotFoundError:
+    warnings.warn(
+        "gliderpy requires matplotlib and cartopy for plotting.",
+        stacklevel=1,
+    )
+    raise
+
+
+if TYPE_CHECKING:
+    import pandas as pd
+
+from pandas_flavor import register_dataframe_method
+
+@register_dataframe_method
+
```

Comment:
Let's remove this blank link to make it easier to read that we are decorating this specific function. We will add more in the future and blank lines after a decorator can be confusing when there are many.
```suggestion
```